nglview/staticlab/package.json

```diff
@@ -94,8 +94,14 @@
   "jupyterlab": {
     "extension": "lib/jupyterlab-plugin",
     "outputDir": "nglview-js-widgets/labextension",
+    "sharedPackages": {
+      "@jupyter-widgets/base": {
+        "bundled": false,
+        "singleton": true
+      }
+    },
```

Comment:
why are those new lines here? I think they already exist to support jupyterlab with iypwidget 8.

Reply:
oh, this is in `staticlab`.